libs/util/app-config/src/lib/app-config.ts

```diff
@@ -96,7 +97,7 @@ export function loadAppConfig() {
         parsed,
         'global',
         ['geonetwork4_api_url'],
-        ['proxy_path'],
+        ['proxy_path', 'metadata_language'],
```

Comment:
Can we implement a check here to make sure that the provided language is valid? So:
1/ if not a valid code, stop the app and warn the user (e.g. `fra` instead of `fre`, `de` instead of `ger`)
2/ if not lowercase, force it to be

Reply:
otherwise all search requests will fail silently

Reply:
Good point, thanks, the `lowerCaser()` is added, still have to add the check.

Reply:
I've started with https://github.com/geonetwork/geonetwork-ui/pull/264 regarding this.